oep_models.py

```diff
@@ -162,8 +162,4 @@ class ResPowerPlantRegister(Base):
 
```

Comment:
you can install flake8 and pylint locally to run the tests locally :)

(`pip install pylint flake8`)